spec/requests/api/v1/moves_controller_create_spec.rb

```diff
@@ -252,12 +252,6 @@
 
       let(:errors_422) do
         [
-          {
-            'title' => 'Unprocessable entity',
-            'detail' => "Date can't be blank",
-            'source' => { 'pointer' => '/data/attributes/date' },
-            'code' => 'blank',
-          },
```

Comment:
This is the only test that broke as a result of making `date` optional for `proposed` moves. What's happening with this spec is that an invalid status value is passed as a property; that then means that a move instance is created but does not get the invalid status assigned. Instead the status value defaults to the first one defined in the enumerator, which is `proposed`. Hence as a proposed move the missing date attribute is not considered an error. Hmm.. as I'm writing this I'm thinking I might instead update the test to check that missing `Date from` throws an error instead 🤔